IOS/SwiftSeedProject/Podfile

```diff
@@ -6,5 +6,6 @@ target 'SwiftSeedProject' do
   use_frameworks!
 
   # Pods for SwiftSeedProject
+  pod 'Bond'
```

Comment:
Can you specify the version of Bond we are using? Since iOS usually include breaking-changes, it's safer to define a specific version and update only if required.